src/activy/connectStrava.py

```
@@ -65,48 +64,58 @@ def connectStravaStage4(device, templates):
-            tryCheckStage(device, 4, templates)
-        except:
-            tryCheckStage(device, 402, templates)
+        stage = tryCheckStage(device, ["4", "4o401", "401or4o402", "402or4o403"], templates, retries=50)
+        if stage[0] == "4o401":
+            connectStravaStage401(device, templates)
+        elif stage[0] == "401or4o402":
             connectStravaStage402(device, templates)
+        elif stage[0] == "402or4o403":
+            connectStravaStage403(device, templates)
+
+    except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)